README.md

```diff
@@ -248,6 +248,7 @@ Additional lists you might find useful:
 - [Feedback plugin](https://github.com/dereuromark/cakephp-feedback) - Allow visitors to send quick and easy feedback incl. a screenshot via sidebar form.
 - [Flash plugin](https://github.com/dereuromark/cakephp-flash) - More powerful flash messages for your application.
 - [Hashid plugin](https://github.com/dereuromark/cakephp-hashid) - Allows to use hashids to not expose the database ids to the user.
+- [Heartbeat plugin](https://github.com/orca-services/cakephp-heartbeat/) - Monitor the hearbeat of your application (e.g. whether the database is available and up-to-date).
```

Comment:
How about linking the 3.x relevant branch directly? 
https://github.com/orca-services/cakephp-heartbeat/tree/cakephp-3.x

Reply:
We dont do that. as this would mean we always have to modify this all the time.